packages/@n8n/db/src/entities/test-case-execution.ee.ts

```diff
@@ -54,4 +54,10 @@ export class TestCaseExecution extends WithStringId {
 
 	@JsonColumn({ nullable: true })
 	metrics: TestCaseRunMetrics;
+
+	@JsonColumn({ nullable: true })
+	inputs: Record<string, GenericValue> | null;
+
+	@JsonColumn({ nullable: true })
+	outputs: Record<string, GenericValue> | null;
```

Comment:
`GenericValue` is `string | object | number | boolean | undefined | null`

- `undefined` is allowed even though this is JSON? 
- `object` is quite wide and allows `Date`, `RegExp`, `Function`, etc. Are these all expected?

Reply:
Like this:

```ts
type JsonValue = string | boolean | null | number | JsonObject | JsonValue[];
type JsonObject = { [key: string]: JsonValue; };
```

Reply:
Thanks for the suggestion. I updated the types.